src/containers/Profile/Profile.js

```diff
@@ -141,18 +141,27 @@ class Profile extends Component {
               <Input s={12} l={6} placeholder="First Name" label="Firstname" name="firstName" onChange={this.handleInputChange} value={this.state.firstName} />
               <Input s={12} l={6} placeholder="Last Name" label="Lastname" name="lastName" onChange={this.handleInputChange} value={this.state.lastName}/>
               <Input s={12} l={6} placeholder="Contact" label="Contact" name="contact" onChange={this.handleInputChange} value={this.state.contact}/>
-              <Input s={12} l={6} type='select' label="Gender" name="gender" onChange={this.handleInputChange} value={this.state.gender}>
-                <option disable='disable'>Select </option>
-                <option value='male'>Male</option>
-                <option value='female'>Female</option>
-              </Input>
+
```

Comment:
i modified this part of the code because react-materialize dropdown suddenly stopped working and the whole app was breaking